modules/light-clients/08-wasm/keeper/snapshotter.go

```diff
@@ -99,7 +99,7 @@ func (ws *WasmSnapshotter) RestoreExtension(height uint64, format uint32, payloa
 	return errorsmod.Wrapf(snapshot.ErrUnknownFormat, "expected %d, got %d", ws.SnapshotFormat(), format)
 }
 
-func restoreV1(ctx sdk.Context, k *Keeper, compressedCode []byte) error {
+func restoreV1(_ sdk.Context, k *Keeper, compressedCode []byte) error {
```

Comment:
Could we remove the sdk.Context entierly here?

Reply:
ah yes, removed here [remove sdk.Context and height args from WasmSnapshotter functionality](https://github.com/cosmos/ibc-go/pull/7880/commits/862fe3f4be1f1d668aaaa1ac95677fb9d0960ea2).

note this also ended up removing an unused `height` argument, changing the `RestoreExtension` method signature

Reply:
jk, actually [restore height arg because thats an iface impl](https://github.com/cosmos/ibc-go/pull/7880/commits/b3c9914af3266fc4dfdd5f3f8d65fecc744bce69)